e2e_test/sink/t2.slt

```diff
@@ -9,6 +9,7 @@ CREATE SINK s2 FROM mv2 WITH (
     connector = 'mysql',
     endpoint = '127.0.0.1:3306',
     user = 'root',
+    password = '123456'
     database = 'test',
     table = 't2'
```

Comment:
we are testing with sink node, so should set `connector = 'jdbc'` here?

Reply:
`t2.slt` and `t3.slt` are mysql sinks that exists long before this PR.